jboss-eap-xp1-openjdk11-openshift.json

```diff
@@ -0,0 +1,123 @@
+{
+    "kind": "List",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "jboss-eap-xp1-openjdk11-openshift",
+        "annotations": {
+            "description": "ImageStream definition for JBoss Enterprise Application Platform expansion pack with OpenJDK 11",
+            "openshift.io/provider-display-name": "Red Hat."
+        }
+    },
+    "items": [
+        {
+            "kind": "ImageStream",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "jboss-eap-xp1-openjdk11-openshift",
+                "annotations": {
+                    "openshift.io/display-name": "JBoss EAP XP with OpenJDK 11",
+                    "openshift.io/provider-display-name": "Red Hat",
+                    "version": "1.0"
+                }
+            },
+            "labels": {
+                "xpaas": "1.0"
+            },
+            "spec": {
+                "tags": [
+                    {
+                        "name": "latest",
+                        "annotations": {
+                            "description": "The latest released build of JBoss EAP XP",
+                            "iconClass": "icon-eap",
+                            "tags": "builder,eap-xp,javaee,java,jboss,hidden",
+                            "supports": "eap-xp:1,javaee:8,java:11",
+                            "version": "latest",
+                            "openshift.io/display-name": "JBoss EAP XP with OpenJDK 11"
+                        },
+                        "referencePolicy": {
+                            "type": "Local"
+                        },
+                        "from": {
+                            "kind": "DockerImage",
+                            "name": "registry.redhat.io/jboss-eap-7/eap-xp1-openjdk11-openshift-rhel8:latest"
+                        }
+                    },
+                    {
+                        "name": "1.0",
+                        "annotations": {
+                            "description": "JBoss EAP XP Builder Image",
+                            "iconClass": "icon-eap",
+                            "tags": "builder,eap-xp,javaee,java,jboss,hidden",
```

Comment:
why is it `hidden`?

Reply:
This is historical. In OCP 3.x, and especially OpenShift Online, there was a great deal of concern around having too many items in the display when templates / imagestreams were installed. So anything having the tag hidden, doesn't display in the web UI, but is still viewable when you use oc get imagestreams etc.

It might make sense to drop hidden from the builder image, but keep the runtime image hidden perhaps?